ergotree-interpreter/src/eval/scontext.rs

```diff
@@ -186,4 +338,98 @@ mod tests {
         };
         assert_eq!(eval_out::<AvlTreeData>(&expr, &ctx), avl_tree_data);
     }
+
+    #[test]
+    fn eval_getvar_from_input_type_mismatch() {
+        let ctx = prepare_getvar_from_input_context(0i16, 1u8, 123i32);
+        let result = eval_out::<Option<Value>>(&create_method_call(SType::SByte, 0, 1), &ctx);
+        assert_eq!(result, None);
+    }
+
+    #[test]
+    #[should_panic(expected = "getVar: type mismatch")]
+    fn eval_getvar_type_mismatch() {
+        let ctx = prepare_getvar_context(1u8, 123i32);
+        eval_out::<Option<Value>>(&create_getvar_method_call(SType::SByte, 1), &ctx);
+    }
+
+    proptest! {
+    #![proptest_config(ProptestConfig::with_cases(64))]
+
+         #[test]
```

Comment:
rustfmt doesn't work inside proptest macros unfortunately, can you manually fix some of the indent issues?